File: src/main/java/com/github/thaynarasilvapinto/SimuladorBanco/SimuladorBancoApplication.java
package com.github.thaynarasilvapinto.SimuladorBanco;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;


@SpringBootApplication
@EnableJpaAuditing
public class SimuladorBancoApplication{

	public static void main(String[] args) {
		SpringApplication.run(SimuladorBancoApplication.class, args);
	}

}

File: simulador-banco-web/src/main/kotlin/com/github/thaynarasilvapinto/web/SimuladorBancoApplication.kt
package com.github.thaynarasilvapinto.web

import com.github.thaynarasilvapinto.repositories.config.RepositoryConfig
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import

@SpringBootApplication
@Configuration
@Import(RepositoryConfig::class)
@ComponentScan(basePackages = ["com.github.thaynarasilvapinto"])
open class SimuladorBancoApplication

fun main(args: Array<String>) {
    SpringApplication.run(SimuladorBancoApplication::class.java, *args)
}